assets/app/views/pods.html

```diff
@@ -8,10 +8,58 @@
           <em>{{emptyMessage}}</em>
         </div>
       </div>      
-      <div style="margin-bottom: 10px;" ng-repeat="pod in pods">
-        <h3>{{pod.metadata.name}} ({{pod.status.phase}})</h3>
+      <div style="margin-bottom: 10px;" ng-repeat="pod in pods | orderObjectsByDate : true" class="tile">
+        <h2>{{pod.metadata.name}}</h2>
+        <div>
+          <span style="margin-right: 5px;">Status:  {{pod.status.phase}}</span>
+          <span ng-switch="pod.status.phase">
+            <span ng-switch-when="Succeeded" class="fa fa-check text-success" aria-hidden="true" style="margin-right: 5px;"></span>
+            <span ng-switch-when="Terminated" class="fa fa-times text-danger" aria-hidden="true" style="margin-right: 5px;"></span>
+            <span ng-switch-when="Pending" class="fa fa-refresh fa-spin" aria-hidden="true" style="margin-right: 5px;"></span>
+            <span ng-switch-default class="fa fa-refresh text-success" aria-hidden="true" style="margin-right: 5px;"></span>
+          </span>
+        </div>
+        <div>
+          Node: {{pod.status.host || 'unknown'}} <span ng-if="pod.status.hostIP && pod.status.host != pod.status.hostIP">({{pod.status.hostIP}})</span>
+        </div>
+        <div>
+          IP on node: {{pod.status.podIP || 'unknown'}}
+        </div>
+        <div>
+          Restart policy:
+          <span ng-if="pod.spec.restartPolicy.always">
+            always <span ng-if="pod.status.info.POD">(restarted {{pod.status.info.POD.restartCount}} times)</span>
+          </span>
+          <span ng-if="pod.spec.restartPolicy.onFailure">
+            on failure <span ng-if="pod.status.info.POD">(restarted {{pod.status.info.POD.restartCount}} times)</span>
+          </span>          
+          <span ng-if="pod.spec.restartPolicy.never">
+            never
```

Comment:
could they have changed the spec to restart never, yet it previously had restarted?